packages/ozone-api/ozone-api-upload/src/ozone-api-upload.ts

```diff
@@ -250,6 +258,37 @@ export class UploadFileRequest implements XMLHttpRequestLike {
                 return null
         });
     }
+    /**
+     * alias to send method.
+     * @param {FormData} file
+     * @param {string} parentTenantId
+     * @return {Promise<null>}
+     */
+    importOrganisation(file: FormData, parentTenantId: string): Promise<null> {
+
+        return Promise.resolve(OzoneConfig.get())
+            .then((config) => {
+                this.config = config;
```

Comment:
Do we really have to store config in a global since we call `OzoneConfig.get()` at every call ?
This comment goes also for `uploadFile` function. The scope of the config could be reduced to these functions instead of a global.